mirar/database/transactions/update.py

```diff
@@ -0,0 +1,35 @@
+"""
+Module to update database entries
+"""
+from sqlalchemy import text, update
+
+from mirar.database.base_table import BaseTable
+from mirar.database.constraints import DBQueryConstraints
+from mirar.database.engine import get_engine
+
+
+def _update_database_entry(
```

Comment:
should this be returning the sequence keys?

Reply:
No. In the DB updater processor, you don't return anything. In the inserter, if there is a duplicate, you return the key from a select call.